plugin.py

```diff
@@ -128,40 +136,49 @@ def is_valid_for_view(self, view: sublime.View) -> bool:
     def _handle_log_message_notification(self, payload: CopilotPayloadLogMessage) -> None:
         pass
 
+    @notification_handler(NTFY_PANEL_SOLUTION)
+    def _handle_panel_solution_notification(self, payload: CopilotPayloadPanelSolution) -> None:
```

Comment:
Why do we need this? I mean cannot we handle all this in the `NTFY_PANEL_SOLUTION_DONE`?

Reply:
It would take quite a long time. Solutions are sent one by one once Copilot figures out one of them.